src/main/java/uk/ac/cam/cl/dtg/segue/auth/exceptions/DuplicateAccountException.java

```diff
@@ -23,6 +23,7 @@
  */
 public class DuplicateAccountException extends SegueDatabaseException {
     private static final long serialVersionUID = 7415512495026915539L;
+    public static final String DEFAULT_MESSAGE = "An account with that email address already exists.";
```

Comment:
Should this be `private` instead? I nearly made it in one of the Constants files, but this seemed a more sensible home.

Reply:
```suggestion
    private static final String DEFAULT_MESSAGE = "An account with that email address already exists.";
```
If `DEFAULT_MESSAGE` is only accessed by this class it should be `private`. As it currently isn't being accessed outside the class I would suggest that the change be made now.

I agree that it should be placed in this class as it is related to this class.